README.md

```diff
@@ -40,7 +40,10 @@ Environment variables:
  - `USER_COUNT` which is set to `1` by default
  - `HUMAN_ACTION_MIN_DELAY` which is set to `7` by default. The minimum amount of seconds between 2 human actions.
  - `HUMAN_ACTION_MAX_DELAY` which is set to `15` by default. The maximum amount of seconds between 2 human actions.
- - `AUTHENTICATION_STRATEGY` which is set to `basic` by default (you can use `lemonldap` instead)
+ - `AUTHENTICATION_STRATEGY` which is set to `basic` by default (you can use `lemonldap`, `oidc` or `pkce` instead)
+ - `OIDC_CLIENT` oidc client to use for `oidc` and `pkce` authentication strategies
+ - `OIDC_CALLBACK` url to redirect to after a successful login with `oidc` and `pkce` authentication strategies
+ - `PKCE_CODE_CHALLENGE_METHOD` which is set to `S256` by default. Which should not be changed at the moment.
```

Comment:
can we remove this? given the spec I think it's useless to make it configurable, and we do not handle it in the code to generate the hash